canonical-websites/www.ubuntu.com
# Moving /engage/kubernetes-case-study to /engage/k8s-allan-gray

## Done

* Moved /engage/kubernetes-case-study to /engage/k8s-allan-gray, fixing #4514

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure `/engage/kubernetes-case-study` is redirected
